public/editpost.php

```diff
@@ -90,7 +90,7 @@ class="w-full"
                     <div class="textarea-counter text-right" data-textarea-id="commentTextarea"></div>
 
                     <div class="flex gap-2">
-                        <img id="preview-loading-icon" src="$loadingIconSrc" style="opacity: 0;" width="16" height="16" alt="Loading..." class="w-4 h-4">
+                        $loadingIcon
```

Comment:
This icon doesn't seem to be centered correctly.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/7ba501c5-a3ff-4a3a-a1b6-d78807274d7e)

Reply:
Updated

![Screenshot 2024-01-09 at 18 43 45](https://github.com/RetroAchievements/RAWeb/assets/1018784/fded3b35-60c8-4182-9a9e-6f2f3fc056cb)